src/SFA.DAS.AssessorService.Application.Api/Controllers/ExternalApi/EpaBatchController.cs

```diff
@@ -0,0 +1,198 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.AssessorService.Api.Types.Models;
+using SFA.DAS.AssessorService.Api.Types.Models.ExternalApi.Epas;
+using SFA.DAS.AssessorService.Api.Types.Models.Standards;
+using SFA.DAS.AssessorService.Application.Api.Middleware;
+using SFA.DAS.AssessorService.Application.Api.Properties.Attributes;
+using SFA.DAS.AssessorService.Application.Api.Validators.ExternalApi.Epas;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using NotFound = SFA.DAS.AssessorService.Domain.Exceptions.NotFound;
+
+namespace SFA.DAS.AssessorService.Application.Api.Controllers.ExternalApi
+{
+    [Authorize]
```

Comment:
There is a test somewhere which check that each controller has an Authorize attribute - will that test check the external API as well?

Reply:
Yeah it will check it within the ControllerAuthorizeAttributeTests checks. Only PingController is allowed not to have one